Fix OpenAPI spec drift, added missing query parameters, and ensured consistent parameter ordering across list endpoints (#27938)
directus/directus: .changeset/carone-hying-wilding.md

---
'@directus/specs': patch
---

> Fixed OpenAPI spec drift, added missing query parameters, and ensured consistent parameter ordering across list endpoints.

Reviewer: ```suggestion
Fixed OpenAPI spec drift, added missing query parameters, and ensured consistent parameter ordering across list endpoints
```

> Fixed OpenAPI spec drift on query parameters across list endpoints. Parameter lists are now consistently ordered to match the [query parameters documentation](https://directus.com/docs/guides/connect/query-parameters).

Reviewer: We can probably generalize this a bit more as it is also adding missing params
```suggestion
Fixed OpenAPI spec drift and ensured query parameters are consistently ordered.
```
